README.md

```diff
@@ -19,13 +19,24 @@ helm repo update
 Charts are packaged and released with [`cr`](https://github.com/helm/chart-releaser)
 when the `develop` branch is merged into `master`.
 
-To make releasing easier two scripts have been included:
-
-- `.github/release-vector-version.sh` queries [vectordotdev/vector](https://github.com/vectordotdev/vector)
-for the latest release and updates the `vector` chart's default image. This is
-convenient when updating the chart after a Vector release.
-- `.github/release-changelog.sh` pulls the current `vector` chart version and
-uses `git-cliff` to update the [CHANGELOG.md](CHANGELOG.md). Run this to generate
-the final commit merged into `develop` before merging `develop` into `master`.
-This script requires [`yq`](https://github.com/mikefarah/yq) and
+To make releasing easier two scripts are utilized in the steps below.
+
+1. `$ .github/release-vector-version.sh`
+    - This queries [vectordotdev/vector](https://github.com/vectordotdev/vector)
+for the latest release and updates the `vector` chart's default image.
+    - This is convenient when updating the chart after a Vector release.
+
+2. Commit the changes generated from step 1. This needs to be a
+[conventional commit](https://www.conventionalcommits.org/).
+    - E.g. "feat(vector): Bump Vector to v0.29.0"
```

Comment:
I think Jesse/myself have been doing `fix` for patch releases and `feat` for major/minor releases, fwiw.

Reply:
Ah that is good to know. I will use that on my branch for the patch.